libs/ui/sidenav/src/lib/sidenav-trigger/sidenav-trigger.component.html

```diff
@@ -43,7 +43,7 @@
   cdkConnectedOverlayHasBackdrop
   cdkConnectedOverlayBackdropClass="ts-sidenav-panel-backdrop"
   [cdkConnectedOverlayOrigin]="triggerItem"
-  [cdkConnectedOverlayOpen]="panelType !== 'drawer' && isPanelOpen"
+  [cdkConnectedOverlayOpen]="panelType === 'popout' && isPanelOpen"
```

Comment:
Your change means that the panel type has to be `popout` to trigger the overlay open. Are you sure you mean that?

Reply:
@atlwendy As I understand for `panelType = undefined` we should not have overlay.
Please correct me If I am wrong.

![image](https://user-images.githubusercontent.com/16876059/113431812-fceab980-93e4-11eb-8b59-9f1a5e7a2431.png)

In template we have 2 declarations of overlay (drawer, popout):
```
[cdkConnectedOverlayOpen]="panelType === 'drawer' && isPanelOpen"
[cdkConnectedOverlayOpen]="panelType === 'popout' && isPanelOpen"
```

`export type TS_SIDENAV_PANEL_TYPES = 'popout'  | 'drawer'  | undefined;`

Reply:
Okay! I assumed that you checked the demo site and the symptom has been fixed, right?
Also want to make sure down the road, if there're more panel types added, we'd also update this accordingly. It is fine currently.

Reply:
I have checked it locally and here: [chromatic](https://5f0ca4e61af3790022cad2fe-ryqmmjvabf.chromatic.com/?path=/story/components-navigation-sidenav--basic)